Modules/SubModule_ASL/xASL_wrp_RealignASL.m

```diff
@@ -109,9 +141,15 @@ function xASL_wrp_RealignASL(x,bSubtraction)
 MeanRadius = 50; % typical distance center head to cerebral cortex (Power et al., NeuroImage 2012)

```

Comment:
@BeatrizPadrela Apparently, this needs adaptation in some cases. Something for the future :)